sigridci/sigridci/report.py

```diff
@@ -76,7 +76,9 @@ def getSummaryText(self, feedback, options):
             return f"✅  You wrote maintainable code and achieved your Sigrid objective of {target}"
         elif status == ObjectiveStatus.IMPROVED:
             return f"↗️  You improved your code's maintainability towards your Sigrid objective of {target}"
+        elif status == ObjectiveStatus.UNCHANGED:
+            return "⏸️️  You didn't improve your code, but it also didn't get any worse"
```

Comment:
The phrase "but it also didn't get any worse" suggests that the code isn't good to begin with, so it sounds very negative to me. For instance, you have a target of 3.5, you're currently at 3.4, and the current Sigrid CI analysis determines the rating did not change. Then I think the user doesn't deserve the "... any worse" bit. Here's a more neutral alternative:
```suggestion
            return "⏸️️  You didn't improve your code, and consequently the rating did not change"
```

Reply:
Fair point. There is another thing with this message: This is the only one that doesn't mention what your objective actually is. That's purely for aesthetics, it doesn't look nice when the title word-wraps. But ideally we would find a way to have a similar meaning as this suggestion, but in a way that *also* allows us to put the objective in there.